platform/email/platform-email-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,7 +18,12 @@
            xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
            http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
+    <reference id="encryptionService"
+        interface="ddf.security.encryption.EncryptionService"
+        availability="mandatory"/>
```

Comment:
i think this is default

Reply:
![image](https://user-images.githubusercontent.com/37838624/89590189-47477680-d815-11ea-82d3-f8f007666c46.png)

Reply:
I'm not sure if it's necessary or not - I just copy pasted from another location in the codebase